Backend.ts

```diff
@@ -50,7 +50,8 @@ export class Backend<T> {
 		start?: isoly.Date | isoly.DateTime,
 		end?: isoly.Date | isoly.DateTime,
 		limit?: number,
-		continuation?: string
+		continuation?: string,
+		alternateDateTimeProperty?: string
```

Comment:
This name seems a bit convoluted, can't you name it something shorter?

Reply:
Alternatives: `timeProperty`, `time`, `property`, `indexBy`, `by`

You could also do
`property = "created"` on the argument. So you don't need the tertiary statement, this also does not make it an **alternative** property, just a property.

Reply:
You could also do
`alternateDateTimeProperty = "created"` on the argument. So you don't need the tertiary statement, this also does not make it an **alternative** property, just a property.

Reply:
added default and shorted it to `dateTimeProperty`